frontend/src/common/interfaces/index.ts

```diff
@@ -0,0 +1,2 @@
+export type { IRegisterPayload } from 'healthcare-shared/common/interfaces';
```

Comment:
It should be in the `frontend/src/common/interfaces/register/index.ts`

and in the `frontend/src/common/interfaces/index.ts` should be this:
```
export * from './register'
```